### PR TITLE
[PL-51075] SMP 0.14.8 patch RN

### PR DESCRIPTION
**Pull request by brian-f-harness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: SMP 0.14.8 patch RN
* Jira/GitHub Issue numbers (if any): PL-51075
* Preview links/images (Internal contributors only): https://663e310ae261e5008476ab32--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition#may-10-2024-patch-version-0148

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by vcalasansh — DISMISSED** [review]
LGTM

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -82,6 +82,49 @@ If you don't use Helm to upgrade Harness Self-Managed Enterprise Edition, follow
...
+| Platform Service | 1.12.0 |
+| Access Control Service | 1.33.2 |
+| Delegate | 24.02.82203 |
+| Change Data Capture | 1.5.3 |
+| STO Core | 1.83.8 |
+| Test Intelligence Service | 1.8.1 |
+| NG UI | 1.7.6 |
+| LE NG | 1.1.0 |
+
+**Alternative air gap bundle download method**
```
Comment (douglas-j-bothwell) [inline release-notes/self-managed-enterprise-edition.md:105]: Maybe make this a level-4 heading?

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -82,6 +82,49 @@ If you don't use Helm to upgrade Harness Self-Managed Enterprise Edition, follow
...
+  "gs://smp-airgap-bundles/harness-0.14.8/ci_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.14.8/ff_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.14.8/platform_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.14.8/sto_images.tgz" \
+  .
+```
+
+### Early access feature
+
+- Currently, in artifact and manifest triggers, when a build is removed from the artifact server, it is automatically deleted from the cache of the trigger. As a result, if this build is uploaded again, the trigger will use it once more to initiate the underlying pipeline.
```
Comment (douglas-j-bothwell) [inline release-notes/self-managed-enterprise-edition.md:124]:
```suggestion
- The current behavior for artifact and manifest triggers is that, when a build is removed from the artifact server, it is automatically deleted from the cache of the trigger. As a result, if this build is uploaded again, the trigger will use it once more to initiate the underlying pipeline.
```
Optional. Not sure this wording is necessarily better but it does remove one comma.

File: release-notes/self-managed-enterprise-edition.md [hunk 3]
```diff
@@ -82,6 +82,49 @@ If you don't use Helm to upgrade Harness Self-Managed Enterprise Edition, follow
...
+
+   You can now change this behavior for artifact and manifest triggers to guarantee a trigger is activated only once per collected build. Even if the build is deleted and subsequently pushed again, the trigger will not fire a second time. This option is behind the feature flag `CDS_DISABLE_POLLED_KEYS_EVICTION_FOR_ARTIFACT_TRIGGER_POLLING_DOCUMENT`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. (PIPE-18763)
```
Comment (douglas-j-bothwell) [inline release-notes/self-managed-enterprise-edition.md:126]:
```suggestion
   You can now change this behavior for artifact and manifest triggers to guarantee that a trigger is activated only once per collected build. Even if the build is deleted and subsequently pushed again, the trigger will not fire a second time. This option is behind the feature flag `CDS_DISABLE_POLLED_KEYS_EVICTION_FOR_ARTIFACT_TRIGGER_POLLING_DOCUMENT`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. (PIPE-18763)
```

**Review by douglas-j-bothwell — APPROVED** [review]
Approved with a few optional word-nerd sugs.

File: release-notes/self-managed-enterprise-edition.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md]: Nice catch @douglas-j-bothwell . Updated for all versions.

**Pull request merged**